### PR TITLE
Update Code Caching for Devs post section on service worker cache.

### PR DESCRIPTION
**Pull request by wanderview**

### CODE REVIEW

**Comment by LeszekSwirski** [comment]
This looks great, thanks!

**Comment by LeszekSwirski** [comment]
@RReverser should we add an "updated" note somewhere around here?

**Comment by RReverser** [comment]
> I've added the `updated` value to the frontmatter.
> 
> >

Added support for such field in #420, let's merge that first.

**Comment by wanderview** [comment]
> Added support for such field in #420, let's merge that first.

It looks like that merged.  Can someone with permissions merge this now?

File: src/blog/code-caching-for-devs.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ author: '[Leszek Swirski](https://twitter.com/leszekswirski), cache smasher'
 avatars:
   - leszek-swirski
 date: 2019-04-08 13:33:37
+updated: 2020-06-09 13:33:37
```
Comment (RReverser) [inline src/blog/code-caching-for-devs.md:7]: Please update date to 16-06 (and probably remove time, that doesn't look like an actual time anyway?).

Comment (wanderview) [reply src/blog/code-caching-for-devs.md]: Done.  I updated to "2020-06-16".  I removed the time.

**Comment by wanderview** [comment]
> Seems like you unintentionally committed a vim `.swp` file.

Thanks.  Removed.

**Pull request merged**